docker/image_types.yaml

```diff
@@ -17,54 +17,87 @@
   depends_on:
     - 'base-image'
 
-'coverage-builder':
-  tag: 'builders/coverage'
-  context: 'fuzzers/coverage'
-  dockerfile: 'fuzzers/coverage/builder.Dockerfile'
+'dispatcher-image':
+  tag: 'dispatcher-image'
+  context: 'docker/dispatcher-image'
+  depends_on:
+    - 'base-image'
+
+'.{fuzzer}-builder':
```

Comment:
Are the dot prefixes for hiding makefile rules? If yes, consider adding them at makefile rule generation. This file should only encode the properties and dependencies of the image types in a generic way

Reply:
Done